src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountVerificationService.java

```diff
@@ -60,13 +77,34 @@ private void sendMediaUsersForVerification() {
     }
 
     /**
-     * Method that gets all media users who have not verified their account in 365 days.
+     * Method that gets all AAD amin users who last signed in at least 76 days (by default) ago, and gets all
+     * IDAM users who last signed in at least 118 days (by default) ago.
+     * Then send their details on to publication services to send them a notification email.
+     */
+    private void notifyAadAdminAndIdamUsersToSignIn() {
+        // TODO: this is a placeholder. Needs to be updated with call to Publication services for notifying the users.
+        Stream.of(
+                userRepository.findAadAdminUsersByLastSignedInDate(aadAdminAccountSignInNotificationDays),
+                userRepository.findIdamUsersByLastSignedInDate(idamAccountSignInNotificationDays))
+            .flatMap(Collection::stream)
+            .forEach(user -> log.info("Remind user to sign in: " + user.getEmail()));
+    }
+
+    /**
+     * Method that gets all media users who have not verified their account (default to 365 days), and AAD admin and
+     * IDAM users who have not signed in to their account (default to 90 days for AAD admin and 128 days for IDAM).
      * Account service handles the deletion of their AAD, P&I user and subscriptions.
      */
     private void findAccountsForDeletion() {
-        userRepository.findVerifiedUsersByLastVerifiedDate(mediaAccountDeletionDays).forEach(user -> {
-            log.info(accountService.deleteAccount(user.getEmail()));
-        });
+        Stream.of(
+                userRepository.findVerifiedUsersByLastVerifiedDate(mediaAccountDeletionDays),
+                userRepository.findAadAdminUsersByLastSignedInDate(aadAdminAccountDeletionDays),
+                userRepository.findIdamUsersByLastSignedInDate(idamAccountDeletionDays))
+            .flatMap(Collection::stream)
+            .forEach(
+                user -> log.info(accountService.deleteAccount(user.getEmail(),
```

Comment:
Strickly not part of this ticket but feels like might as well do it now.

Can we update this log statement to use the P&I logger rather than the SLF4J one?